menu/menu_setting.c

```diff
@@ -2025,6 +2025,9 @@ static void setting_get_string_representation_toggle_gamepad_combo(
       case INPUT_TOGGLE_HOLD_START:
          strlcpy(s, msg_hash_to_str(MENU_ENUM_LABEL_VALUE_HOLD_START), len);
          break;
+      case INPUT_TOGGLE_DOWN_SELECT:
+         strlcpy(s, "Down + Select", len);
```

Comment:
Should likely be localized via `msg_hash_to_str(MENU_ENUM_LABEL_VALUE_DOWN_SELECT)` or similar.